frontend/src/action-creator.sync.js

```diff
@@ -118,35 +113,32 @@ export const updateWorkspaceDetail = workspaceDetail => ({ type: `${UPDATE}/${WO
 export const WORKSPACE_LOADED = `${WORKSPACE}/Loaded`
 export const setWorkspaceLoaded = () => ({ type: `${SET}/${WORKSPACE_LOADED}` })
 
-const MEMBER = 'Member'
-export const WORKSPACE_MEMBER = `${WORKSPACE}/${MEMBER}`
-export const WORKSPACE_MEMBER_LIST = `${WORKSPACE_MEMBER}/List`
-export const setWorkspaceMemberList = workspaceMemberList => ({ type: `${SET}/${WORKSPACE_MEMBER_LIST}`, workspaceMemberList })
-// INFO - CH - 2023-11-22 - WORKSPACE_MEMBER_ADD is used by api function postWorkspaceMember()
-// but nothing is bound to it
-export const WORKSPACE_MEMBER_ADD = `${WORKSPACE_MEMBER}/${ADD}`
-// INFO - CH - 2023-11-22 - WORKSPACE_MEMBER_ADD is used by api function deleteWorkspaceMember()
-// but nothing is bound to it
-export const WORKSPACE_MEMBER_REMOVE = `${WORKSPACE_MEMBER}/${REMOVE}`
-export const WORKSPACE_MEMBER_UPDATE = `${WORKSPACE_MEMBER}/${UPDATE}`
-export const ADD_WORKSPACE_MEMBER = `${ADD}/${WORKSPACE_MEMBER}`
+const ROLE = 'Role'
+export const USER_ROLE = `${USER}/${ROLE}`
+export const USER_ROLE_LIST = `${USER_ROLE}/List`
+export const SET_USER_ROLE_LIST = `${SET}/${USER_ROLE_LIST}`
+export const setUserRoleList = userRoleList => ({ type: SET_USER_ROLE_LIST, userRoleList })
+export const USER_ROLE_UPDATE = `${USER_ROLE}/${UPDATE}`
```

Comment:
the "update" should be before, it's the same as "WORKSPACE_MEMBER_ADD" that you removed
That const doesn't seem to be used anyway, you can remove it

Reply:
const is deleted